git-config/src/file/git_config.rs

```diff
@@ -187,6 +274,7 @@ impl<'event> GitConfig<'event> {
             // These two are supposed to share the same scope and override
             // rather than append according to git-config(1) documentation.
             if let Ok(xdg_config_home) = env::var("XDG_CONFIG_HOME") {
+                // TODO make sure the path exists, as git does
```

Comment:
Is this something to address in this PR or is it for later? Both is fine, your call.

Reply:
actually nothing to address, can be NULL: https://github.com/git/git/blob/master/config.c#L1953

Reply:
Thanks, sounds reasonable, especially since we also ignore non-existing paths.